bin/run-browser-tests-local

```diff
@@ -3,34 +3,11 @@
 # DOC: Run the browser tests locally. Requires browser test env already running.
 
 source bin/lib.sh
-export BASE_URL=http://localhost:9999
+# Disable screenshots for local tests because it will be use browser on current
+# machine which can be different from the browser used by browser-test docker
+# image and produce different screenshots.
 export DISABLE_SCREENSHOTS=true
-export TEST_USER_AUTH_STRATEGY=fake-oidc
-export TEST_USER_LOGIN=testuser
-export TEST_USER_PASSWORD=anotsecretpassword
-# The display name returned by test_oidc_provider.js is <username>@example.com.
-export TEST_USER_DISPLAY_NAME=testuser@example.com
-
-START_TIME=$(date +%s)
-DEADLINE=$(($START_TIME + 500))
-
-if ! output="$(node -v)"; then
```

Comment:
Can we keep the node requirement just in the place that requires it?  It's not common to run -local so as is we'll require people to install it when they may never need it.

Reply:
This is the place that requires it. When we call npm few lines down - it requires node present. When engineers run this script from run-browser-test - the file is executed from within docker and docker environment has node. So engineers don't need to have node unless they run -local.

Reply:
Those two scripts also require and use node. Because they use npm. Previously they didn't do any checks for node presence and assumed that node/npm was always present.

Reply:
The browser-test image (which is based on [playwright image](https://github.com/civiform/civiform/blob/main/browser-test/playwright.Dockerfile#L2)) comes with node preinstalled.